src/backend/services/users-api/src/app/routes.py

```diff
@@ -79,49 +68,49 @@ def getUsers():
 
 
 @app.route('/create', methods=['POST'])
-def createUser():
+def create_user():
     data = request.get_json()
     try:
         first_name = data["first_name"]
         last_name = data["last_name"]
         password = data["password"]
-        umnetID = data["umnetID"]
+        umnetID = data["umnetID"].upper()
         public_key = data["public_key"]
-
-    except Exception as e:
-        return jsonify(error=FailureReturnString.INCORRECT_PAYLOAD.value), HttpCode.BAD_REQUEST.value
+    
+    except KeyError as e:
+        raise IncorrectPayloadException()
 
     user = {
-        "first_name": first_name,
-        "last_name": last_name,
-        "umnetID": umnetID.upper(),
-        "password": password,
-        "public_key": public_key
+        "first_name" : first_name,
+        "last_name" : last_name,
+        "umnetID" : umnetID,
+        "password" : password,
+        "public_key" : public_key
     }
 
     try:
         mongo.db.users.insert_one(user)
+
     except Exception as e:
-        return jsonify(error=FailureReturnString.DATABASE_VERIFICATION_FAILURE.value, err=str(e)), HttpCode.BAD_REQUEST.value
+        raise DatabaseVerificationException(str(e))
 
     return jsonify(
-        success=True,
-    ), 201
-
+        success = True,
+        message= 'User ' + first_name + ' ' + last_name + ' added successfully! :)'
```

Comment:
I removed the message field in one of my PRs, cant remember if it's the create account or edit. I really don't think we need it and would like to leave it out! I updated the relevant tests to check success instead of the message contents